content/en/docs/apidocs-mxsdk/apidocs/design-properties.md

```diff
@@ -135,23 +204,251 @@ This is how the **Dropdown** design property appears:
+* `property` (required if any of the options uses a CSS variable): A CSS property.
 
-{{% alert color="warning" %}}
-Name your design property and its options carefully. Those names cannot be changed easily when there are apps already using them. If you want to rename a design property which is already being used in an app, see the [Renaming Design Properties](#old-names) section below.
-{{% /alert %}}
+When the selected option defines a class, that class will be applied to the widget. When the selected option defines a variable, it will be applied to the specified `property`.
 
-#### 4.1.2 Toggle-Specific Fields
+Here is an example of a **Colorpicker** design property using classes:
+
+```js
+{
+    "name": "Background color",
+    "type": "Colorpicker",
+    "description": "Description of Background color Property",
+    "options": [
+        {
+            "name": "Red",
+            "class": "backgroundColorRed"
+        },
+        {
+            "name": "Green",
+            "class": "backgroundColorGreen"
+        },
+        {
+            "name": "Blue",
+            "class": "backgroundColorBlue"
+        }
+    ]
+}
+```
 
-When a type of design property is **Toggle** it should contain a `class` field on the top level of its property definition. This field defines an arbitrary class name to be applied if the option is toggled on for a widget. In the example above, the class to be applied is `hereMyClass`.
+Here is an example of a **Colorpicker** design property using CSS variables:
 
-#### 4.1.3 Dropdown-Specific Fields
+```js
+{
+    "name": "Background color",
+    "type": "Dropdown",
+    "property": "background-color",
+    "description": "Description of Background Color Property",
+    "options": [
+        {
+            "name": "Red",
+            "variable": "--color-red"
+        },
+        {
+            "name": "Green",
+            "variable": "--color-green"
+        },
+        {
+            "name": "Blue",
+            "variable": "--color-blue"
+        }
+    ]
+}
+```
+
+Note: if you can not provide a value for the preview, it is recommended to instead use a **Dropdown** design property. If at a later point in time you can provide the preview, you can always change the type from a **Dropdown** to a **Colorpicker**.
+
+#### 4.1.5 ToggleButtonGroup-Specific Fields
+
+Design properties of type **ToggleButtonGroup** have the following additional fields:
+
+* `multiSelect` (optional): When `true`, the user can set multiple options at the same time, meaning the CSS classes of all selected options will be applied to the widget. This cannot be `true` if any of the options uses a CSS variable. When `false` or if the field is not present, only a single option can be selected.
```

Comment:
Maybe we should mention that a user shouldn't change this value once it is set, because that would cause problems